driver-scala/src/main/scala/org/mongodb/scala/connection/package.scala

```diff
@@ -41,6 +41,21 @@ package object connection {
    */
   type SocketSettings = com.mongodb.connection.SocketSettings
 
+  /**
+   * This setting is only applicable when communicating with a MongoDB server or a Key Management Service
+   * using the synchronous variant of `com.mongodb.MongoClient` or `ClientEncryption`.
+   *
+   * This setting is furthermore ignored if:
+   * - the communication is via `com.mongodb.UnixServerAddress` (Unix domain socket).
+   * - a `StreamFactoryFactory` is `MongoClientSettings.Builder.streamFactoryFactory` configured.
```

Comment:
- The text mentions KMS, which is no longer relevant. The text in `ProxySettings.java` is differs from this one.
- The list is not rendered as a list by Scaladoc.